### PR TITLE
Ssl

### PR DESCRIPTION
**Pull request by matrinox**

### CODE REVIEW

**Review by senorprogrammer — COMMENTED** [review]
I'm not qualified to comment on this project or these changes, I don't know enough about them, or the impact these will have on this codebase.

Seems like something that should be opened as a PR against the parent project and have the maintainers look at?

File: mix.exs [hunk 1]
```diff
@@ -35,7 +35,7 @@ defmodule Exq.Mixfile do
   defp deps do
     [
       { :uuid, ">= 1.1.0" },
-      { :redix, ">= 0.5.0"},
+      { :redix, ">= 0.6.1" },
```
Comment (AaronBetterOfficeApps) [inline mix.exs]: Is this our version of remix? if so I think we should add something like `0.6.1-ld` so that it won't conflict with the office `0.6.1`

Comment (matrinox) [reply mix.exs]: They reference the commit sha so it will match the correct version. Are you saying it's a conflict in reading it?

**Comment by matrinox** [comment]
@senorprogrammer done https://github.com/akira/exq/pull/285

**Pull request closed without merging**